components/molecules/Pagination/Pagination.cy.tsx

```diff
@@ -0,0 +1,270 @@
+import { mount } from "cypress/react18";
+import Pagination from ".";
+import { SearchParamsContext } from "next/dist/shared/lib/hooks-client-context.shared-runtime";
+import I18NextClientProvider from "@/contexts/i18next";
+
+describe("<Pagination>", () => {
+  it("renders to page", () => {
+    const searchParams = new URLSearchParams({ page: "2" });
+    const limit = 30;
+    const offset = 30;
+    const page = 2;
+    const total = 90;
+
+    mount(
+      <I18NextClientProvider locale="en">
+        <SearchParamsContext.Provider value={searchParams}>
+          <Pagination limit={limit} offset={offset} page={page} total={total} />
+        </SearchParamsContext.Provider>
+      </I18NextClientProvider>
+    );
+
+    cy.viewport(1440, 900);
```

Comment:
suggestion: if you're using a viewport repeatedly in tests, it can be defined in the test config: https://docs.cypress.io/api/commands/viewport#Set-viewport-in-the-test-configuration